pkg/cortex/serve/cortex_internal/lib/model/tfs.py

```diff
@@ -113,7 +113,13 @@ def __init__(self, address: str):
             {}
         )  # maps the model ID to the model metadata (signature def, signature key and so on)
 
-        self.channel = grpc.insecure_channel(self.address)
+        # set maximum/receive transmission sizes to 256 MB each
```

Comment:
Is there an option to set it to unlimited? If not, perhaps 1GB or 10GB?

Reply:
Good idea. Checked if it's possible and it is indeed. One has to set each field to `-1`.